k-distribution/include/kframework/builtin/domains.md

```diff
@@ -533,7 +538,7 @@ The set with zero elements is represented by `.Set`.
 An element of a `Set` is constructed via the `SetItem` operator.
 
 ```k
-  syntax Set ::= SetItem(KItem)               [function, functional, hook(SET.element), klabel(SetItem), symbol]
+  syntax Set ::= SetItem(KItem)               [function, functional, hook(SET.element), klabel(SetItem), symbol, injective]
```

Comment:
Why is this a `function` instead of a constructor? We would get injectivity for free if it was a constructor.